client/trino-client/src/main/java/io/trino/client/ClientSession.java

```diff
@@ -315,27 +322,87 @@ private Builder(ClientSession clientSession)
             compressionDisabled = clientSession.isCompressionDisabled();
         }
 
+        public Builder withServer(URI server)
```

Comment:
Remove the `with` prefix from all these methods.

Reply:
ack! removed all the "with" prefix.